3-enrich/scala-hadoop-enrich/src/test/scala/com.snowplowanalytics.snowplow.enrich.hadoop/good/CljTomcatSendgridEventSpec.scala

```diff
@@ -0,0 +1,187 @@
+/*
+ * Copyright (c) 2012-2014 Snowplow Analytics Ltd. All rights reserved.
```

Comment:
ah, sorry! Intellij hides that copyright notice at the top so I completely forgot about it. Fixed